src/types.ts

```diff
@@ -258,4 +258,5 @@ export interface Cache<Data = any> {
   get(key: Key): Data | null | undefined
   set(key: Key, value: Data): void
   delete(key: Key): void
+  clear(key: Key): void
```

Comment:
With this signature, what would be the difference between `delete` and `clear`? iirc, the cache used here is pretty much a Map? those have a `clear` method that takes no arguments, which removes all `key-value` pairs from the Map.

Reply:
> With this signature, what would be the difference between `delete` and `clear`? iirc, the default cache used here is a Map? those have a `clear` method that takes no arguments, which removes all `key-value` pairs from the Map.
> 
> Edit:
> 
> Relevant code https://github.com/vercel/swr/blob/main/src/utils/config.ts#L41 - I think it would be good to just have `clear(): void`

My bad! You are right. I fixed this with commit c54ce94.